pkgs/applications/terminal-emulators/zutty/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, freetype
+, wafHook
+, python3
+, libXmu
+, glew-egl
+, ucs-fonts
+}:
+stdenv.mkDerivation rec {
+  pname = "zutty";
+  version = "0.14";
+
+  src = fetchFromGitHub {
+    owner = "tomscii";
+    repo = "zutty";
+    rev = "bc6e0893e91bc0ffd08cb7d15727973de450c3d6";
```

Comment:
??

Reply:
Oh, sorry. I blindly took the output of nix-prefetch-github as I do when hacking together some temp stuff. Bad habit. I'll change it to the version variable. Thanks for catching it!

Reply:
Fixed. Was there anything else I should address?